site/package.json

```diff
@@ -48,7 +48,26 @@
     "solid-js": "^1.7.2",
     "solid-start": "^0.2.26",
     "undici": "^5.15.1",
-    "vite-plugin-solid-markdown": "^0.0.6"
+    "vite-plugin-solid-markdown": "^0.0.6",
+    "@webcontainer/api": "1.0.0-alpha.1",
+    "xterm": "^5.2.0",
+    "xterm-addon-fit": "^0.7.0-beta.2",
+    "split.js": "^1.6.5",
+    "@codemirror/autocomplete": "^6.0.4",
+    "@codemirror/commands": "^6.0.1",
+    "@codemirror/lang-css": "^6.0.0",
+    "@codemirror/lang-html": "^6.1.0",
+    "@codemirror/lang-javascript": "^6.0.1",
+    "@codemirror/lang-json": "^6.0.0",
+    "@codemirror/lang-markdown": "^6.0.0",
+    "@codemirror/language": "^6.2.0",
+    "@codemirror/language-data": "^6.1.0",
+    "@codemirror/search": "^6.0.0",
+    "@codemirror/state": "^6.1.0",
+    "@codemirror/theme-one-dark": "^6.0.0",
+    "@codemirror/view": "^6.0.3",
+    "codemirror": "^6.0.1",
```

Comment:
![image](https://github.com/Flouse/ckb-academy/assets/1297478/dd830d48-0da1-41c4-b559-999ff77efb88)

### Does `codemirror` support adjusting the colors of the above code blocks?

Reply:
I know where the problem is, and I will handle it